src/components/input/GeoInput/index.js

```diff
@@ -202,6 +216,10 @@ export default class GeoInput extends React.PureComponent {
         const polygons = this.getPolygons(value);
         const options = this.getAllGeoOptions(geoOptionsByRegion);
 
+        const polygonSupportIndex = accessibleFeatures.findIndex(f => f.key === 'polygon_support_geo');
```

Comment:
We should create constants for feature name somewhere

Reply:
yes we should

Reply:
In constants?